src/cmd/cgogen.go

```diff
@@ -81,17 +81,16 @@ var arrayTypes = map[string]string{
 }
 
 //Imports used in this code file
-var importDefs [](*ast.GenDecl)
+var importDefs []*ast.GenDecl
 
 //types that will be replaced by handles
 var handleTypes map[string]string
 var returnVarName = "____error_code"
 var returnErrName = "____return_err"
-var dealOutStringAsGostring = true
-var get_package_path_from_file_name = true
```

Comment:
why removed ?

Reply:
In the complete code there is no modification or use of these variables.

Reply:
because you delete in PR the code the makes use of `get_package_path_from_file_name` , that's why .

Reply:
because it will always be `true` always a module comes from one or several files, and also after go 1.11.x that almost manages it only with the` format`. So I see it unnecessary

Reply:
ok